helm/charts/determined/templates/master-config.yaml

```diff
@@ -166,6 +172,9 @@ stringData:
     {{- if .Values.resourcePools}}
     resource_pools:
       {{- toYaml .Values.resourcePools | nindent 6}}
+      {{- if .Values.genai.version }}
+      - pool_name: a100
+      {{- end }}
```

Comment:
this probably must be removed? we need users to configure RPs in `.Values.resourcePools` and can't just add an extra resource pool (what if it needs default pod specs and so on).